components/org.wso2.bfsi.consent.management.extensions/src/main/java/org/wso2/bfsi/consent/management/extensions/manage/builder/ConsentManageBuilder.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.consent.management.extensions.manage.builder;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.bfsi.consent.management.common.util.ConsentManagementConstants;
+import org.wso2.bfsi.consent.management.extensions.common.ConsentExtensionUtils;
+import org.wso2.bfsi.consent.management.extensions.internal.ConsentExtensionsDataHolder;
+import org.wso2.bfsi.consent.management.extensions.manage.ConsentManageHandler;
+
+/**
+ * Builder class for consent manage handler.
+ */
+public class ConsentManageBuilder {
+
+    private static final Log log = LogFactory.getLog(ConsentManageBuilder.class);
+    private ConsentManageHandler consentManageHandler = null;
+
+    public void build() {
+
+        String handlerConfig = (String) ConsentExtensionsDataHolder.getInstance().getConfigurationService()
+                .getConfigurations().get(ConsentManagementConstants.MANAGE_HANDLER);
+        consentManageHandler = (ConsentManageHandler) ConsentExtensionUtils.getClassInstanceFromFQN(handlerConfig);
+
+        log.debug("Manage handler loaded successfully");
+    }
+
+    public ConsentManageHandler getConsentManageHandler() {
+        return consentManageHandler;
```

Comment:
Shall we follow a different pattern in this class? with the current approach, to get the manage handler we need write 3 lines

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/4/commits/98992be82ffa4483d8c6f51e4dce81ee11bb9d3b